### PR TITLE
Add vaccine dose distribution copy.

### PR DESCRIPTION
**Pull request by mikelehen**
image

### CODE REVIEW

File: src/common/metrics/vaccinations.tsx [hunk 1]
```diff
@@ -65,13 +65,24 @@ function renderStatus(projections: Projections): React.ReactElement {
   const peopleVaccinated = formatInteger(info.peopleVaccinated);
   const percentVaccinated = formatPercent(info.ratioVaccinated, 1);
 
+  let distributedText = <Fragment />;
+  if (info.dosesDistributed && info.ratioDosesAdministered) {
+    distributedText = (
+      <Fragment>
+        This this means that {locationName} has administered{' '}
```
Comment (ghop02) [inline src/common/metrics/vaccinations.tsx:72]: Two this's here.  Also, "this means that" feels a bit unnecessary - being pedantic, it doesn't really imply the % usage of total supply, it only really implies the total doses administered.  Could this just be `{locationName} has administered`?

File: src/common/metrics/vaccinations.tsx [hunk 2]
```diff
@@ -65,13 +65,24 @@ function renderStatus(projections: Projections): React.ReactElement {
...
+        {formatPercent(info.ratioDosesAdministered)} of their supply of{' '}
```
Comment (ghop02) [inline src/common/metrics/vaccinations.tsx:73]: current supply?

**Review by ghop02 — COMMENTED** [review]
couple small copy notes!

File: src/common/metrics/vaccinations.tsx (hunk 1 above)
Comment (mikelehen) [reply src/common/metrics/vaccinations.tsx]: Gah, good catch on "This this". 😞 

"This means that" was bugging me too, so I removed it. But it is the copy that Debbie had approved.  But I think it's probably an improvement.

File: src/common/metrics/vaccinations.tsx (hunk 2 above)
Comment (mikelehen) [reply src/common/metrics/vaccinations.tsx]: Yeah, I was thinking about that too.  Done.

**Review by ghop02 — APPROVED** [review]
LGTM!

**Pull request merged**